xds/src/main/java/io/grpc/xds/internal/security/certprovider/FileWatcherCertificateProviderProvider.java

```diff
@@ -99,6 +101,9 @@ private static Config validateAndTranslateConfig(Object config) {
     configObj.certFile = checkForNullAndGet(map, CERT_FILE_KEY);
     configObj.keyFile = checkForNullAndGet(map, KEY_FILE_KEY);
     configObj.rootFile = checkForNullAndGet(map, ROOT_FILE_KEY);
+    if (map.containsKey(SPIFFE_TRUST_MAP_FILE_KEY)) {
+      configObj.spiffeTrustMapFile = checkForNullAndGet(map, SPIFFE_TRUST_MAP_FILE_KEY);
```

Comment:
`JsonUtil.getString()` instead of `checkForNullAndGet()`. You just checked that the value was present.